core/spec/models/spree/variant_spec.rb

```diff
@@ -263,8 +263,8 @@
     end
 
     context 'variant' do
-      it 'should return options text' do
-        expect(variant.exchange_name).to eql 'Foo Type: Foo'
+      it 'should return options text with name' do
+        expect(variant.exchange_name).to eql variant.name + ' - Foo Type: Foo'
```

Comment:
I would like to see the expectation explicitly defined rather than the method call to `variant.name`.  Something like:

```Ruby
expect(variant.exchange_name).to eql 'Fancy Shirt - Foo Type: Foo'
```